### PR TITLE
feat: page-structure-static

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Static pages
2. Folder structure
3. Basic components: SearchBox, Note

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
liked that you have created an array for pages and using useLocation

File: src/components/Note/Note.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { BsPin } from "react-icons/bs";
+import { BiArchiveIn, BiTrashAlt } from "react-icons/bi";
+import { MdOutlineLabel, MdOutlineColorLens } from "react-icons/md";
```
Comment (t007rushi) [inline src/components/Note/Note.jsx]: create separate file for this

File: src/components/SearchBox/SearchBox.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FaSearch, FaFilter } from "react-icons/fa";
+import "./search-box.css";
+
+function SearchBox(props) {
+  console.log(props);
```
Comment (t007rushi) [inline src/components/SearchBox/SearchBox.jsx:6]: console to remove

File: src/page-layout/Main/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import Archive from "../../pages/Archive/Archive";
+import Home from "../../pages/Home/Home";
+import Login from "../../pages/Login/Login";
+import PageNotFound from "../../pages/PageNotFound/PageNotFound";
+import Profile from "../../pages/Profile/Profile";
+import Trash from "../../pages/Trash/Trash";
```
Comment (t007rushi) [inline src/page-layout/Main/Main.jsx]: try single line export

File: src/page-layout/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Link, NavLink } from "react-router-dom";
+import "./navbar.css";
+
+function Navbar() {
+  const getActiveLinkStyle = ({ isActive }) => ({
```
Comment (t007rushi) [inline src/page-layout/Navbar/Navbar.jsx:6]: liked that you separate out active styling

File: src/pages/PageNotFound/PageNotFound.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function PageNotFound() {
+  return (
+    <div className="text-center text-md">
+      PageNotFound ----- Broken link h yar!!
```
Comment (t007rushi) [inline src/pages/PageNotFound/PageNotFound.jsx]: ```h yar``` ??? use proper messaging

**Review by t007rushi — APPROVED** [review]
Look for suggestion to push before merge

File: src/pages/PageNotFound/PageNotFound.jsx (hunk 5 above)
Comment (ankit9015) [reply src/pages/PageNotFound/PageNotFound.jsx]: Thanks, will update these suggestions

**Pull request merged**